### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Dargmuesli**
- Moves modules to `drinks_scanner_display` and resources to their subfolder with own fonts folder
- Adds VENV settings
- Adds mail_pw and config.py go .gitignore

### CODE REVIEW

**Comment by Dargmuesli** [comment]
WIP until #35 is cosed. Needs testing.

**Comment by Dargmuesli** [comment]
I've also built an Python3 upgrade branch on top of this. When this here is merged, I'll open the PR for that.

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,4 @@ venv-*/
 .ideaconfig.py
 config.py
 ldap_pw
-mail_pw
\ No newline at end of file
+mail_pw
```
Comment (cfstras) [inline .gitignore]: isn't mail_pw obsolete now that we have that moved to config.py?

**Comment by cfstras** [comment]
please rebase or resolve conflicts

**Comment by Dargmuesli** [comment]
Done, but this is still untested.

File: .gitignore (hunk 1 above)
Comment (Dargmuesli) [reply .gitignore]: Yes, but that's not in master yet.

**Comment by cfstras** [comment]
all broken...

**Pull request merged**